### ISSUE TITLE
Build fails with JDK 9

### ISSUE THREAD

**Issue by apjanke** [relaxng/jing-trang#223]
The `jing-trang` build fails for me with JDK 9. This is on macOS 10.13.3.

```
$ git clone https://github.com/relaxng/jing-trang
Cloning into 'jing-trang'...
remote: Counting objects: 24179, done.
remote: Total 24179 (delta 0), reused 0 (delta 0), pack-reused 24179
Receiving objects: 100% (24179/24179), 27.99 MiB | 1.77 MiB/s, done.
Resolving deltas: 100% (13140/13140), done.
[~/tmp/build-jing-trang]
$ cd jing-trang
[~/tmp/build-jing-trang/jing-trang on ⇄ master]
$ ./ant
Buildfile: ./build.xml

check-modules:

modules:

check-modbuild:

modbuild:
     [xslt] Processing /Users/janke/tmp/build-jing-trang/jing-trang/modules.xml to /Users/janke/tmp/build-jing-trang/jing-trang/modbuild.xml
     [xslt] Loading stylesheet /Users/janke/tmp/build-jing-trang/jing-trang/build.xsl

jar:

init:
    [mkdir] Created dir: /Users/janke/tmp/build-jing-trang/jing-trang/build

mod.util.compile-main:
    [mkdir] Created dir: /Users/janke/tmp/build-jing-trang/jing-trang/build/mod/util/classes/main
    [javac] Compiling 34 source files to /Users/janke/tmp/build-jing-trang/jing-trang/build/mod/util/classes/main
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.5
    [javac] error: Source option 1.5 is no longer supported. Use 1.6 or later.
    [javac] error: Target option 1.5 is no longer supported. Use 1.6 or later.

BUILD FAILED
/Users/janke/tmp/build-jing-trang/jing-trang/build.xml:56: The following error occurred while executing this line:
/Users/janke/tmp/build-jing-trang/jing-trang/modbuild.xml:1731: Compile failed; see the compiler error output for details.

Total time: 0 seconds
[✘ ~/tmp/build-jing-trang/jing-trang on ⇄ master ±]
$ java -version
java version "9.0.4"
Java(TM) SE Runtime Environment (build 9.0.4+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.4+11, mixed mode)
[~/tmp/build-jing-trang/jing-trang on ⇄ master ±]
$ javac -version
javac 9.0.4
```

**Comment by georgebina** [comment]
It looks like Java 9 does not support Java 1.5 source and target. Try changing the following lines
https://github.com/relaxng/jing-trang/blob/master/build.xsl#L14
https://github.com/relaxng/jing-trang/blob/master/build.xsl#L15
to use 1.6 instead of 1.5, as the error message suggests.

**Comment by apjanke** [comment]
That works. When I change the source & target to 1.6, the build succeeds and all tests pass. Though with a lot of warnings like the following.

```
mod.schematron.compile-main:
    [javac] Compiling 12 source files to /Users/janke/tmp/build-jing-trang/jing-trang/build/mod/schematron/classes/main
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6
    [javac] warning: [options] source value 1.6 is obsolete and will be removed in a future release
    [javac] warning: [options] target value 1.6 is obsolete and will be removed in a future release
    [javac] warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
    [javac] Note: Some input files use or override a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] 4 warnings
     [copy] Copying 5 files to /Users/janke/tmp/build-jing-trang/jing-trang/build/mod/schematron/classes/main
```

Would you want a PR to change this in the distribution so `jing-trang` supports Java 9 out of the box? Or do you need to preserve Java 1.5 compatibility?

Alternately, I guess I could fuss with the `ant` wrapper script and have it locate a version 8 or earlier JDK, and avoid running under Java 9.

**Comment by sideshowbarker** [comment]
Closing this because I believe I fixed it on the master branch in fce4ba8. But if you find I missed something, please comment here further and we can reopen as needed.
